tests/test_integration_default.py

```diff
@@ -19,7 +19,7 @@ def test_collected(snapshot, actual):
         ),
         "test_not_collected": (
             """
-            def test_not_collected(snapshot):
+            def test_collected1(snapshot):
```

Comment:
This tests the issue by ensuring the `test_collected` is a subset when using `-m parametrized` to select only `test_collected`

Reply:
Did this as an easy test check because it matches the parameterized test above `test_collected`